dockerfile

```
@@ -40,7 +40,7 @@ EXPOSE 16698
 
 # 给予 *.sh 执行权限
 RUN find . -type f \( -name "*.sh" -o -name "*.fish" -o -name "*.zsh" \) -print0 | xargs -0 chmod +x
-RUN find ./path -maxdepth 1 -type f -print0 | xargs -0 chmod +x
+RUN find ./path -maxdepth 1 -type f ! -name 'desktop.ini' ! -iname 'agents.md' -print0 | xargs -0 chmod +x
```

Hadolint (2.15.1)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)